server/controllers/index.ts

```diff
@@ -11,6 +12,7 @@ export const controllers = (services: Services) => {
 
   return {
     dashboardController,
+    ...caseListControllers(services),
```

Comment:
I'm torn between your approach here and having an `assessControllers` and a `referControllers` for the two journeys, the latter of which feels a bit more user friendly when thinking about the different journeys - would it be tricky to split them in this way? This is obviously further complicated by the existence of the `referNewControllers`

Reply:
Yeah I think I agree. Everything is nested by journey unless shared. So this would be `controllers/assess/CaseListController` and the Refer one would be `controllers/refer/caseListController`. I think we can just move `referNewControllers` up a level and rename it `referControllers`?

Reply:
Yeah agreed, makes sense 👍